Lab1.ipynb

```diff
@@ -0,0 +1,310 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#2B\n",
+    "d = stats.lognorm.rvs(s = 1, size = 100000)\n",
+    "fig, ax = plt.subplots(1, 2)\n",
+    "s = 1\n",
+    "x = np.linspace(stats.lognorm.ppf(0.01, s),\n",
+    "                stats.lognorm.ppf(0.99, s), 100)\n",
+    "ax[0].hist(d, 100, density=True)\n",
+    "ax[0].plot(x, stats.lognorm.pdf(x,s),\n",
+    "       'r-', lw=2, alpha = 0.6)\n",
+    "\n",
+    "d2 = stats.lognorm.rvs(s = .5, size = 100000)\n",
+    "s2 = .5\n",
+    "x = np.linspace(stats.lognorm.ppf(.01, s2),\n",
+    "                stats.lognorm.ppf(.99, s2))\n",
+    "ax[1].hist(d2,100, density=True)\n",
+    "ax[1].plot(x, stats.lognorm.pdf(x,s2),\n",
+    "       'r-', lw=2, alpha = 0.6)\n",
+    "plt.show()\n",
+    "\n",
+    "# A lognormal distribution needs x > 1 and s > 1. As s decreases, the width of the probability distribution increases."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 75,
+   "id": "989c40d5-bad7-4092-af04-407ae6243c82",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.014002205573945036\n"
+     ]
+    }
+   ],
+   "source": [
+    "#3\n",
+    "#A: Chose a value of 9\n",
+    "#B: What is the probability that the candidate value of 9 was detected and it was not a reading of background noise?\n",
```

Comment:
In 3, you don't say what parameters you're using for the log-normal background. I can infer from your code that you're using the first of the distributions you plotted in 2B (s=1) but I think it would be helpful to state so explicitly